<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentWidget</class>
 <widget class="QWidget" name="ContentWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>505</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>505</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>561</width>
     <height>420</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>20</y>
     <width>221</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>125</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>30</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>0</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2020</year>
     <month>11</month>
     <day>5</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2020</year>
     <month>11</month>
     <day>5</day>
    </date>
   </property>
   <property name="displayFormat">
    <string>yyyy/M/d H:mm:ss</string>
   </property>
   <property name="timeSpec">
    <enum>Qt::UTC</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCompute">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>400</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>compute</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>360</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonClear">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
